hosted_control_planes/hosted-control-planes-release-notes.adoc

```diff
@@ -0,0 +1,200 @@
+
+* When you use the {mce} console to add a new node pool to an existing hosted cluster, the same version of {product-title} might appear more than once in the list of options. You can select any instance in the list for the version that you want.
+
+* When a node pool is scaled down to 0 workers, the list of hosts in the console still shows nodes in a `Ready` state. You can verify the number of nodes in two ways:
+
+** In the console, go to the node pool and verify that it has 0 nodes.
+** On the command-rline interface, run the following commands:
+
+*** Verify that 0 nodes are in the node pool by running the following command:
++
+[source,terminal]
+----
+$ oc get nodepool -A
+----
+
+*** Verify that 0 nodes are in the cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get nodes --kubeconfig
+----
+
+*** Verify that 0 agents are reported as bound to the cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get agents -A
+----
+
+* When you create a hosted cluster in an environment that uses the dual-stack network, you might encounter the following DNS-related issues:
+
+** `CrashLoopBackOff` state in the `service-ca-operator` pod: When the pod tries to reach the Kubernetes API server through the hosted control plane, the pod cannot reach the server because the data plane proxy in the `kube-system` namespace cannot resolve the request. This issue occurs because in the HAProxy setup, the front end uses an IP address and the back end uses a DNS name that the pod cannot resolve.
+** Pods stuck in `ContainerCreating` state: This issue occurs because the `openshift-service-ca-operator` cannot generate the `metrics-tls` secret that the DNS pods need for DNS resolution. As a result, the pods cannot resolve the Kubernetes API server.
+To resolve these issues, configure the DNS server settings for a dual stack network.
+
+* On the Agent platform, the {hcp} feature periodically rotates the token that the Agent uses to pull ignition. As a result, if you have an Agent resource that was created some time ago, it might fail to pull ignition. As a workaround, in the Agent specification, delete the secret of the `IgnitionEndpointTokenReference` property then add or modify any label on the Agent resource. The system re-creates the secret with the new token.
+
+* If you created a hosted cluster in the same namespace as its managed cluster, detaching the managed hosted cluster deletes everything in the managed cluster namespace including the hosted cluster. To create a hosted cluster in the same namespace as its managed cluster, see the following situations:
+
+** You created a hosted cluster on the Agent platform by using the {mce} console, and you set the hosted cluster namespace to be the same as managed cluster in the console.
+** You created a hosted cluster by specifying the hosted cluster namespace to be the same as the hosted cluster name.
```

Comment:
You created a hosted cluster via API or CLI by specifying the hosted cluster namespace to be the same as the hosted cluster name.